### PR TITLE
Doc v2 l

### PR DESCRIPTION
**Pull request by jtlap**

### CODE REVIEW

File: include/eve/module/combinatorial/regular/lcm.hpp [hunk 1]
```diff
@@ -46,27 +46,32 @@ namespace eve
...
 //!   **Parameters**
 //!
-//!   `p`, `n`: [ordered arguments](@ref eve::value).
+//!     * `p`, `n`: [ordered arguments](@ref eve::value).
+//!     * `c`: [Conditional expression](@ref conditional_expr) masking the operation.
+//!     * `m`: [Logical value](@ref logical) masking the operation.
 //!
 //!    **Return value**
 //!
 //!    Returns the least common multiple of |p| and |n|.
```
Comment (jfalcou) [inline include/eve/module/combinatorial/regular/lcm.hpp:66]: where are 1. and 2. ???

File: include/eve/module/core/regular/ldexp.hpp [hunk 2]
```diff
@@ -45,34 +45,36 @@ struct ldexp_t : strict_elementwise_callable<ldexp_t, Options, pedantic_option>
...
+//!     * `c`: [Conditional expression](@ref conditional_expr) masking the operation.
+//!     * `n`: [integral value argument](@ref eve::integral_value).
+//!     * `m`: [Logical value](@ref logical) masking the operation.
 //!
 //!    **Return value**
 //!
-//!    The value of \f$\textstyle x 2^n\f$ is returned.
+//!     1. The value of \f$\textstyle x 2^n\f$ is returned.
+//!     2. [The operation is performed conditionnaly](@ref conditional).
+//!     3. Takes extra care to handling limit values.
```
Comment (jfalcou) [inline include/eve/module/core/regular/ldexp.hpp:71]: what about listing them ?

File: include/eve/module/core/regular/lerp.hpp [hunk 3]
```diff
@@ -44,36 +44,32 @@ namespace eve
...
+//!     * `c`: [Conditional expression](@ref conditional_expr) masking the operation.
+//!     * `m`: [Logical value](@ref logical) masking the operation.
 //!
 //!    **Return value**
 //!
-//!    The value of the interpolation (or extrapolation)  between `x` and `y` is returned.
-//!    The call is semantically equivalent to `x+t*(y-x)` but uses fma opportunities.
+//!      1. The value of the interpolation (or extrapolation)  between `x` and `y` is returned.
+//!         The call is semantically equivalent to `x+t*(y-x)` but uses fma opportunities.
+//!      2. `pedantic` version of fma is used internally.
```
Comment (jfalcou) [inline include/eve/module/core/regular/lerp.hpp:69]: Where is 3.

**Review by jfalcou — CHANGES_REQUESTED** [review]

File: include/eve/module/combinatorial/regular/lcm.hpp (hunk 1 above)
Comment (jtlap) [reply include/eve/module/combinatorial/regular/lcm.hpp:66]: done

File: include/eve/module/core/regular/ldexp.hpp (hunk 2 above)
Comment (jtlap) [reply include/eve/module/core/regular/ldexp.hpp:71]: done

File: include/eve/module/core/regular/lerp.hpp (hunk 3 above)
Comment (jtlap) [reply include/eve/module/core/regular/lerp.hpp:69]: done

**Review by jfalcou — APPROVED** [review]

**Pull request merged**